tests/daemon/test-persistent-cache.c

```diff
@@ -954,33 +953,20 @@ static void
 test_persistent_cache_does_not_compute_offset_when_boot_id_is_same (gboolean     *unused,
                                                                     gconstpointer dontuseme)
 {
-  EmerPersistentCache *cache = make_testing_cache ();
-
-  GError *error = NULL;
-  gboolean get_succeeded =
-    emer_persistent_cache_get_boot_time_offset (cache, NULL, &error);
-  g_assert_no_error (error);
-  g_assert_true (get_succeeded);
-  g_assert_true (boot_offset_was_reset ());
-
   gint64 relative_time;
```

Comment:
We should probably wait until just before the second call to `make_testing_cache` to get these timestamps so that we confine the the timestamps in the metadata file to the narrowest possible range. This should benefit both readability and testing fidelity.